### PR TITLE
more docs about configuration

### PR DESCRIPTION
**Pull request by rigazilla**

### CODE REVIEW

File: documentation/asciidoc/topics/ref_configuration_builder.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 = Configuration and Remote Cache Manager APIs
 Use the `ConfigurationBuilder` API to configure {hr_dotnet} client connections and the `RemoteCacheManager` API to obtain and configure remote caches.
 
-.Configuration builder
+.Basic configuration.
```
Comment (oraNod) [inline documentation/asciidoc/topics/ref_configuration_builder.adoc:5]:
```suggestion
.Basic configuration
```

File: documentation/asciidoc/topics/ref_configuration_builder.adoc [hunk 2]
```diff
@@ -35,13 +35,38 @@ namespace simpleapp
...
+// Add a server with specific connection timeouts
+builder.AddServer().Host("127.0.0.1").Port(11222).ConnectionTimeout(90000).SocketTimeout(900);
+// ConfigurationBuilder has fluent interface, options can be appended in chain.
+// Enabling authentication with server name "node0",
+// sasl mech "PLAIN", user "supervisor", password "aPassword", security realm "aRealm"
+builder.Security().Authentication().Enable().ServerFQDN("node0")
+    .SaslMechanism("PLAIN").SetupCallback("supervisor", "aPassword", "aRealm");
+Configuration c = conf.Build();
+----
+.Encryption
```
Comment (oraNod) [inline documentation/asciidoc/topics/ref_configuration_builder.adoc:51]:
```suggestion
----

.Encryption
```

File: documentation/asciidoc/topics/ref_configuration_builder.adoc [hunk 3]
```diff
@@ -35,13 +35,38 @@ namespace simpleapp
...
+builder.AddServer().Host("127.0.0.1").Port(11222);
+// Get configuration builder for encryption
+SslConfigurationBuilder sslBuilder = conf.Ssl();
+// Enable encryption and provide client certificate
+sslBuilder.Enable().ClientCertificateFile("clientCertFilename");
+// Provide server cert if server needs to be verified
+sslBuilder.ServerCAFile("serverCertFilename");
+Configuration c = conf.Build();
+----
+.Cross-site failover
```
Comment (oraNod) [inline documentation/asciidoc/topics/ref_configuration_builder.adoc:64]:
```suggestion
----

.Cross-site failover
```

**Review by oraNod — APPROVED** [review]
Couple of minor nits but LGTM. Thanks, @rigazilla

**Comment by rigazilla** [comment]
merged, thanks for the review @oraNod !

**Pull request merged**